.github/workflows/release-code-freeze.yml

```diff
@@ -116,7 +117,7 @@ jobs:
                   "type": "section",
                   "text": {
                     "type": "mrkdwn",
-                    "text": "• Created a release branch `release/${{ env.RELEASE_VERSION }}` \n • Raised a <https://github.com/Automattic/woocommerce-payments/pull/${{ env.RELEASE_PR_ID }}|Pull Request> to `trunk`\n • Built a <https://github.com/Automattic/woocommerce-payments/actions/runs/$GITHUB_RUN_ID|zip file and ran smoke tests> against it"
+                    "text": "• Created a release branch `release/${{ env.RELEASE_VERSION }}` \n • Raised a <https://github.com/Automattic/woocommerce-payments/pull/${{ env.RELEASE_PR_ID }}|Pull Request> to `trunk`\n • Built a <https://github.com/Automattic/woocommerce-payments/actions/runs/${{ env.GITHUB_RUN_ID }}|zip file and ran smoke tests> against it"
```

Comment:
🤔 `GITHUB_RUN_ID` is always a defined variable based on https://docs.github.com/en/actions/reference/workflows-and-actions/variables

So I think there is something change in env then this value is not picked up properly. My guess is because of the pipe `|` but not 100% sure. 

At least, other places are still using  `$GITHUB_RUN_ID` and it seems fine: 

https://github.com/Automattic/woocommerce-payments/blob/bf55ad3028bbcc6ae2305b668bb238b8ef3d16dd/.github/workflows/build-live-branch.yml#L115

https://github.com/Automattic/woocommerce-payments/blob/bf55ad3028bbcc6ae2305b668bb238b8ef3d16dd/.github/workflows/release-pr.yml#L139

Reply:
My suggestion is to use `${GITHUB_RUN_ID}` (with brackets) rather than `$GITHUB_RUN_ID`

But to ensure that it really works, I think it's worth triggering it manually.

Reply:
It seems that something has changed with either Github Actions or Slack API. 

- This is the test with the current way of using `GITHUB_RUN_ID` p1755055633516079-slack-CGGCLBN58 - it does not work. 
- Using the original approach by @elazzabi makes it work properly p1755056487550659-slack-CGGCLBN58

I am going to revert the original approach by @elazzabi and merge this PR.